pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -87,17 +87,24 @@ let
   removed-providers =
     let
       archived = name: date: throw "the ${name} terraform provider has been archived by upstream on ${date}";
+      license = name: date: throw "the ${name} terraform provider removed from nixpkgs on ${date} because of unclear licensing";
       removed = name: date: throw "the ${name} terraform provider removed from nixpkgs on ${date}";
     in
     lib.optionalAttrs config.allowAliases {
       b2 = removed "b2" "2022/06";
+      checkly = license "checkly" "2022/11";
```

Comment:
I wonder what it need for github to pickup the license correctly: https://github.com/checkly/terraform-provider-checkly/blob/main/LICENSE